elasticsearch/src/main/java/com/gentics/mesh/search/verticle/eventhandler/project/ProjectUpdateEventHandler.java

```diff
@@ -80,7 +80,7 @@ public ProjectUpdateEventHandler(MeshHelper helper, MeshEntities entities, MeshO
 						project.getUuid(),
 						branch.getUuid(),
 						container.getSchemaContainerVersion().getUuid(),
-						type
+						type, null
```

Comment:
Check: do we need the microschemaVersionHash here?

Reply:
Added correct microschemaVersionHash (and verified manually, that re-indexing works correctly)